tests/test_dataset_broker.py

```diff
@@ -32,6 +45,8 @@ def test_produce_consume():
         pytest.skip(
             "Make sure PYTHONPATH is set to where the comet dataset broker is installed."
         )
+    if not has_redis():
+        pytest.skip("Redis is not available and so comet will fail.")
```

Comment:
This is nice! But is it especially difficult to get redis to run on github? https://github.com/zhulik/redis-action

Reply:
Again. I'm basically done with this. This is 100% better than the current Jenkins version, so I'm happy to turn off a few tests to make things work. We can address this in later pull requests.

Reply:
Agree that in theory this is easy, but then it ends up being hours of change, push, build, debug cycles. Plus, I think it's easiest to just install redis into one of the docker images, then you don't have to mess about with bridging the networks.

Reply:
I've actually just enabled these again. Fingers crossed they pass!

Reply:
🍾